packages/runtime/container-runtime/src/summary/summarizer.ts

```diff
@@ -383,4 +384,8 @@ export class Summarizer extends EventEmitter implements ISummarizer {
 		}
 		return this.runningSummarizer.enqueueSummarize(...args);
 	};
+
+	public processOp?(message: ISequencedDocumentMessage) {
```

Comment:
Like it! Have reaching to DM.
That said, can you please double check what Loader versions are in prod, and when I made a change in Container.ts to push ALL ops into runtime? It's likely 6 month old change, and with OWA & WB moving to recent versions of FF I think we will get there quickly, but worth double-checking (and keeping an eye on telemetry) as we could break old clients here...

Reply:
[Move batch processing logic from loader to container runtime](https://github.com/microsoft/FluidFramework/pull/11832) (version 2.0.0-internal.1.2.0).
Looking at Kusto, there are still a couple million sessions of WB in 1.3.4 this past week.

Reply:
We can always expose container runtime option and ask Loop to set it, such that it applies only to Loop scenarios. Kill-bit switch (feature gate) would be required as a safety mechanism.